code/game/turfs/simulated/dungeon/wall.dm

```diff
@@ -16,31 +16,87 @@
 	return
 
 /turf/simulated/wall/solidrock //for more stylish anti-cheese.
-	name = "solid rock"
-	desc = "This rock seems dense, impossible to drill."
 	description_info = "Probably not going to be able to drill or bomb your way through this, best to try and find a way around."
-	icon_state = "bedrock"
-	var/base_state = "bedrock"
+	var/rock_side = "rock_side"
 	block_tele = TRUE
 
-/turf/simulated/wall/solidrock/update_icon()
-	for(var/direction in cardinal)
-		var/turf/T = get_step(src,direction)
-		if(istype(T) && !T.density)
-			var/place_dir = turn(direction, 180)
-			if(!mining_overlay_cache["rock_side_[place_dir]"])
-				mining_overlay_cache["rock_side_[place_dir]"] = image('icons/turf/walls.dmi', "rock_side", dir = place_dir)
-			T.add_overlay(mining_overlay_cache["rock_side_[place_dir]"])
+/turf/simulated/wall/solidrock/New(var/newloc)
+	..(newloc,"bedrock")
 
 /turf/simulated/wall/solidrock/Initialize()
-	icon_state = base_state
+	..()
```

Comment:
Not returning a value in Initialize will cause runtimes

Reply:
Probably should fix upstream.